changelog/1.23.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "1.23.0"
+description: "Released on 09/22/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.23.0.
+
+### Features ✨
+
+- Doctor: [Doctor](https://github.com/cdr/coder-doctor) assesses a Kubernetes
+  cluster to determine its readiness for Coder installation; if there are
+  issues, Doctor provides information on why.
+- Terraform: Coder has released the first of a series of
+  [Terraform scripts](https://github.com/cdr/enterprise-terraform) to facilitate
+  one-click Coder deployment.
+
+- web: added support for [generic applications](../workspaces/applications.md);
+  users can provide images with custom applications, instead of only using
+  applications that have been built into the container image `PATH`.
+- web: added support for DataSpell.
+- web: improved input validation for dev URLs.
+- web: added ability to get access token returned by OIDC providers on login.
+- web: added usage statistics for individual workspaces to the Organizations
+  section of the UI.
+- web: renamed "decommissioned" state to "deleting" when referring to the
+  process of deleting a workspace and freeing up its resources.
+- web: added integrated live chat user so that hosted beta users can reach the
+  Coder support team.
+- web: added in-product changelog.
+- web: added dark mode for the Coder UI.
+- web: added ability to provide a shell command to be run when starting a
+  terminal in Coder.
+- web: updated default workspace resource allocation from 1 core and 1 GB of
+  memory to 4 cores and 4 GB of memory.
+- web: updated audit log to include information about workspace stop actions.
+- web: added ability to specify the Kubernetes service account name when editing
+  a workspace provider.
+- cli: added `run coder update` command; users can now update the Coder CLI
+  directly.
+- api: added ability to use the Coder API to create workspaces.
+- infra: updated logging so that Coder will send an error to `stdout` if unable
+  to write to the in-product audit log.
+- infra: add support for containerd for cached CVMs.
+- infra: updated Coder to always use the latest stable version of JetBrains
+  Projector.
```

Comment:
@jawnsy pointed out we may not "always" version bump before a release. @Emyrk can you confirm?

Reply:
We can omit always?

Reply:
Let's omit always.